code/modules/mob/living/carbon/human/species_types/IPC.dm

```diff
@@ -38,14 +38,9 @@
 /datum/species/ipc/military/check_roundstart_eligible()
 	return FALSE //yes
 
-/datum/species/ipc/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
-	if(chem.type == exotic_blood)
-		H.blood_volume = min(H.blood_volume + round(chem.volume, 0.1), BLOOD_VOLUME_MAXIMUM)
-		H.reagents.del_reagent(chem.type)
-		return 1
-	else
-		H.reagents.del_reagent(chem.type)
-	return FALSE
+/datum/species/ipc/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H) // THIS IS NOW DERELICT CODE BUT SHOULD BE KEPT TO SAFEGUARD IPCS FROM MAXING THE REAGENT LIMIT AND POSSIBLY BREAKING THE UNIVERSE
```

Comment:
Why not implement this into the trait. This isn't futureproof at all

Reply:
because this causes a ton of runtime errors, even when using the proc above (read above this, called remove())

Reply:
(in holder.dm)